theano/configdefaults.py

```diff
@@ -1505,6 +1505,15 @@ def filter_vm_lazy(val):
              EnumStr('regular', 'fast'),
              in_c_key=False)
 
+AddConfigVar('check_stack_trace',
+             "A flag for checking the stack trace during the optimization"
+             " process. The default is not checking the stack trace of any"
+             " optimization. check_all checks if the stack trace is added"
+             " and check_and_skip checks all the stack trace of optimizations"
```

Comment:
Actually, "check_and_skip" inserts the dummy value and does not raises any warnings or exceptions.